impl/src/testing.rs

```diff
@@ -56,3 +96,107 @@ pub fn make_workspace(
 pub fn make_basic_workspace() -> (TempDir, CargoWorkspaceFiles) {
   make_workspace(basic_toml(), Some(basic_lock()))
 }
+
+/** Creates mock responses for fetching crate metadata and source code from a crate registry */
+pub fn mock_remote_crate(name: &str, version: &str, mock_server: &MockServer) -> TempDir {
+  // Crate info mock response
+  mock_server.mock(|when, then| {
+    when.method(GET).path(format!("/api/v1/crates/{}", name));
+    then.status(200).json_body(json!({
+        "crate": {
+            "id": name,
+            "name": name,
+            "versions": [
+                283716
```

Comment:
What is this and where does it come from?

Reply:
it came from the actual data of `cargo-raze` version `0.5.0`. This value simply needs to match the `id` in `versions`.

Reply:
Add a comment? I don't feel strongly about this.

Reply:
Updated